packages/ckeditor5-ui/tests/bindings/clickoutsidehandler.js

```diff
@@ -13,29 +13,38 @@ import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
 
 describe( 'clickOutsideHandler', () => {
 	let activator, actionSpy, contextElement1, contextElement2;
+	let shadowRootContainer, shadowContextElement1, shadowContextElement2;
 
 	testUtils.createSinonSandbox();
 
 	beforeEach( () => {
 		activator = testUtils.sinon.stub().returns( false );
 		contextElement1 = document.createElement( 'div' );
 		contextElement2 = document.createElement( 'div' );
+		shadowRootContainer = document.createElement( 'div' );
+		shadowRootContainer.attachShadow( { mode: 'open' } );
```

Comment:
This will fail when running in IE and other non-shadow-dom-supporting browsers. You might need to detect `attachShadow` and only run these checks if it exists.